pkg/network/msg.go

```diff
@@ -53,6 +53,7 @@ type DeviceAuthResponse struct {
 // DeviceListResponse represents the outgoing list devices command response
 type DeviceListResponse struct {
 	Things []*entities.Thing `json:"devices"`
+	ErrMsg *string           `json:"error"`
```

Comment:
As per commit 2229801ed003112e864cdc4be095e9265f6994e9, this should be `Error` instead :grimacing: